src/settings.cpp

```diff
@@ -398,8 +398,16 @@ size_t BoolSettingDesc::ParseValue(const char *str) const
  */
 void IntSettingDesc::Write_ValidateSetting(const void *object, int32 val) const
 {
-	void *ptr = GetVariableAddress(object, &this->save);
+	this->Validate(val);
```

Comment:
Why do you here choose to keep `Write_ValidateSetting` and in the commit before replace all instances with a call to these two functions?

Reply:
As there are a few places where it is still called. For example in old save game conversion where you do want the new setting to be at least in the valid range, or in LoadSettings where it is used to validate the data in the save game.